mllm/backends/qnn/aot/passes/LLMQuantRecipePass.cpp

```
@@ -0,0 +1,387 @@
+
+#include "mllm/utils/Common.hpp"
+#include "mllm/core/DataTypes.hpp"
+#include "mllm/compile/ir/graph/Op.hpp"
+#include "mllm/compile/ir/linalg/Op.hpp"
+#include "mllm/compile/ir/builtin/Op.hpp"
+#include "mllm/compile/ir/linalg/Attribute.hpp"
+#include "mllm/backends/qnn/aot/passes/AOTCompileContext.hpp"
+#include "mllm/backends/qnn/aot/passes/LLMQuantRecipePass.hpp"
+
+namespace mllm::qnn::aot {
+bool shareQuantSpecSingleInputToSingleOutputAndSetOpQuantAnnoAttr(const ir::IRContext::ptr_t& ctx,
```

Clang (14.0.6)

[error] 2 adjacent parameters of 'shareQuantSpecSingleInputToSingleOutputAndSetOpQuantAnnoAttr' of similar type ('const int &') are easily swapped by mistake

(bugprone-easily-swappable-parameters,-warnings-as-errors)